rustls/src/crypto/ring/tls12.rs

```diff
@@ -367,3 +342,21 @@ impl MessageEncrypter for ChaCha20Poly1305MessageEncrypter {
         Ok(OpaqueMessage::new(msg.typ, msg.version, buf))
     }
 }
+
+fn gcm_nonce(write_iv: &[u8], explicit: &[u8]) -> Iv {
```

Comment:
Should this be called `gcm_iv()` instead of `Iv`? Alternatively, should the `ConnectionTrafficSecrets` type hold a `Nonce` instead of `Iv` for GCM variants?

Reply:
> Should this be called gcm_iv() instead of Iv?

SGTM. Updated.